src/ElmSharp/ElmSharp/ContainerBase.cs

```diff
@@ -0,0 +1,204 @@
+        }
+
+        /// <summary>
+        /// Sets or gets the background color of the Container.
+        /// </summary>
+        /// <remarks>It could be overridden by special child class.</remarks>
+        /// <since_tizen> preview </since_tizen>
+        public override Color BackgroundColor
+        {
+            get => GetBackgroundColor();
+            set => SetBackgroundColor(value);
+        }
+
+        /// <summary>
+        /// Sets or gets the opacity of the widget.
+        /// </summary>
+        /// <remarks>It could be overridden by special child class.</remarks>
+        /// <since_tizen> preview </since_tizen>
+        public override int Opacity
+        {
+            get
+            {
+                //var clipper = Interop.Evas.evas_object_smart_clipped_clipper_get(Handle);
+                Interop.Evas.evas_object_color_get(_clipper, out int r, out int g, out int b, out int a);
+                return a;
+            }
+            set
+            {
+                //var clipper = Interop.Evas.evas_object_smart_clipped_clipper_get(Handle);
+                Interop.Evas.evas_object_color_set(_clipper, 0xff, 0xff, 0xff, value);
+            }
+        }
+
+        /// <summary>
+        /// Adds an object to the Container.
+        /// </summary>
+        /// <param name="child">The oject to be added.</param>
+        /// <since_tizen> preview </since_tizen>
+        public void Add(EvasObject child)
+        {
+            AddChild(child);
+        }
+
+        /// <summary>
+        /// Removes an object from the Container.
+        /// </summary>
+        /// <param name="child">The oject to be removed.</param>
+        /// <since_tizen> preview </since_tizen>
+        public void Remove(EvasObject child)
+        {
+            RemoveChild(child);
+        }
+
+        /// <summary>
+        /// The Container listener that is invoked when a child is added.
+        /// </summary>
+        /// <param name="child">The oject to be added</param>
+        protected override void OnChildAdded(EvasObject child)
+        {
+            base.OnChildAdded(child);
+            Interop.Elementary.elm_box_pack_end(Handle, child.Handle);
+            child.SetClip(_clipper);
+        }
+
+        /// <summary>
+        /// The Container listener that is invoked when a child is removed.
+        /// </summary>
+        /// <param name="child">The object to be removed</param>
+        protected override void OnChildRemoved(EvasObject child)
+        {
+            base.OnChildRemoved(child);
+            Interop.Elementary.elm_box_unpack(Handle, child);
+            Interop.Evas.evas_object_clip_unset(child);
+        }
+
+        /// <summary>
+        /// Creates a widget handle.
+        /// </summary>
+        /// <param name="parent">Parent EvasObject</param>
+        /// <returns>Handle IntPtr</returns>
+        /// <since_tizen> preview </since_tizen>
+        protected override IntPtr CreateHandle(EvasObject parent)
```

Comment:
When an inherited subobject overrides CreateHandle (), all functions associated with _layoutCallback will be expanded, but will be in an inactive state. Is this the intended action? I do not think it is a good structure.